### PR TITLE
Install lessc to fix 9.0 builds

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
+      - npm
```
Comment (moylop260) [inline .travis.yml]: travis has npm installed by default

**Comment by rvalyi** [comment]
aside from npm already being present :+1: LGTM

**Comment by pedrobaeza** [comment]
We can only see the result applying this to any 9.0 branch. Can you make a test PR changing the source of MQT in Travis in any of the regular repos?

**Comment by dreispt** [comment]
Self tests are supposed to prove that. Aren't they enough?

**Comment by pedrobaeza** [comment]
But then you should add 9.0 line to travis

**Comment by dreispt** [comment]
@pedrobaeza Yes, I'll do that.

**Comment by dreispt** [comment]
@rvalyi I wasn't sure it was provided by default. But maybe we can keep to have the dependency explicit.

**Comment by dreispt** [comment]
Fixed.

**Comment by pedrobaeza** [comment]
The change in global doesn't affect, because there's no line that not specify the version explicitly. Please add one.

**Comment by pedrobaeza** [comment]
Sorry, I didn't see first line. All is OK.

:+1:

**Comment by moylop260** [comment]
:+1:

**Comment by rvalyi** [comment]
:+1: @dreispt  I have nothing against letting it explicit (what I'm against is the crappy Odoo webstack itself remember :-p )

**Comment by dreispt** [comment]
Can someone do the honors?

**Comment by pedrobaeza** [comment]
Of course :smiley:

**Comment by guewen** [comment]
Seems this commit breaks some things such as the build of the documentation using https://github.com/OCA/odoo-sphinx-autodoc.

See https://travis-ci.org/OCA/connector/jobs/89212615

If I understand correctly, npm creates a `node_modules` directory which is mingled with the addons-path. Maybe should we use the global option `-g` or use the `--prefix` option to move this directory elsewhere (disclaimer: I don't know very well `npm`, just googled a bit)?

**Comment by pedrobaeza** [comment]
Or maybe blacklist `node_modules` directory in odoo-sphinx-autodoc?

**Comment by guewen** [comment]
> Or maybe blacklist node_modules directory in odoo-sphinx-autodoc?

`odoo-sphinx-autodoc` is not limited to use in mqt with travis, so no, it is not a good idea to add coupling between it and mqt. Also this issue may affect other tools, so it is better if we can fix it directly in travis.

I see 2 possibilities: either we move the other repositories elsewhere than in $HOME (subdirectory) or we ensure that we have only valid addons-path folders in $HOME and then we move `node_modules` elsewhere. I think the first one is cleaner, do you see any problem with that?

**Comment by dreispt** [comment]
I like the first option; don't see any reason not to implement it.

**Comment by pedrobaeza** [comment]
+1 for the first option also

**Comment by guewen** [comment]
Working on it: #289

**Pull request merged**